SECURITY.md

```diff
@@ -0,0 +1,175 @@
+if they can be exploited in a release.
+
+TODO: any other repos with releases? TODO: what about knative-sandbox?
+
+No support is provided for archived repos.
+
+# Reporting a Vulnerability
+
+Please email the
+[Knative Security Committee](mailto:knative-security@googlegroups.com) with
+confidential vulnerability reports. Someone from the committee will get back to
+you within 3 business days. Accepted reports will be fixed within 90 calendar
+days via a minor release and/or one or more patch release(s) for minor versions
+that are both affected and supported at that point.
+
+We ask that you keep reports confidential for the full 90 days, or until the
+committee releases you from the confidentiality, whichever comes first, as we
+coordinate disclosures with Knative partners who may need some time after the
+vulnerability is addressed in Knative in order to update to their systems.
+
+TODO: security isn't a listed working group, receiving security reports is
+described as a function of Steering, and there's not "security committee" listed
+anywhere. So what is Security and who's on it?
+https://knative.dev/community/contributing/working-groups/working-groups/
+https://knative.dev/community/contributing/steering-committee/
+
+TODO: GPG or other tools for ensuring confidentiality of the report?
+
+## What's in a report?
+
+Ideal reports are in English and include a description of the vulnerability, a
+proof-of-concept exploit demonstrating the vulnerability, and a statement of
+what this vulnerability puts at risk.
+
+The security committee may make a best-effort attempt to address reports that do
+not meet these standards.
+
+## Coordinated Vulnerability Disclosure
+
+Knative Security engages with partners, including vendors that host
+Knative-based products, to responsibly disclose newly discovered
+vulnerabilities.
+
+For more information on CVD, please review the information provided in the
+following links:
+
+[Guidelines and Practices for Multi-Party Vulnerability Coordination and Disclosure](https://www.first.org/global/sigs/vulnerability-coordination/multiparty/FIRST-Multiparty-Vulnerability-Coordination-latest.pdf)
+
+[The CERT Guide to Vulnerability Disclosure](https://resources.sei.cmu.edu/asset_files/SpecialReport/2017_003_001_503340.pdf)
+
+[PSIRT Services Framework](https://www.first.org/standards/frameworks/psirts/FIRST_PSIRT_Services_Framework_v1.0.pdf)
+
+# Vulnerability Rewards
+
+Knative itself has no vulnerability report reward program.
+
+Some vulnerability reports may be eligible for rewards through programs from
+vendors, such as those with products based on Knative. Reporters should comply
+with program rules to receive these rewards; the Knative community will not
+submit reports for rewards on the reporter's behalf.
+
+Vendors with relevant reward programs:
+
+- [Google](https://www.google.com/about/appsecurity/reward-program/index.html)
+
+TODO: others? remove list of reward programs for vendor-neutrality?
```

Comment:
I'd be fine with leaving it.  Do these programs have "double dipping" policies?

Reply:
I gave it a quick skim and didn't see anything in the Google VRP about this.

I also took a look through https://knative.dev/docs/knative-offerings/ and while I found places to report security bugs for most of them, I didn't see any other mentions of a rewards program, so double-dipping may be a moot point (at least for now).